bleep-core/src/scala/bleep/GenBloopFiles.scala

```diff
@@ -328,6 +328,27 @@ object GenBloopFiles {
         case _                                            => bloop.config.Tag.Library
       }
 
+    val sourceGenerators = {
+      val sg = explodedProject.`source-generator-scripts`.values.map { sc =>
+        if (!build.scripts.contains(sc.scriptName)) {
+          sys.error(s"Unknown script ${sc.scriptName} referenced by ${crossName.value}")
+        }
+        val codegenPath = buildPaths.generatedSourcesDir(crossName)
+        val scriptCall = List("bleep", sc.scriptName.value)
```

Comment:
to increase the chance of this working, we should grab the absolute path to `bleep` used to run this. You can see that done in `SetupIde#run()`. Try to extract that into it's own thing and reference it both places.

Reply:
Cool, I wanted to ask about that 👍